DESCRIPTION

```diff
@@ -44,3 +44,4 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Config/testthat/edition: 3
 VignetteBuilder: knitr
+Additional_repositories: https://p3m.dev/cran/2024-02-02/
```

Comment:
CRAN acknowledges this field but doesn't use it. `install.packages()` does not use it. `remotes::install_*()` *does* use it, but `pak::pak()` does not (yet) :shrug:

This PPPM repo is pinned to the current arrow release version. Alternatively, you can instead get the latest pre-release version of arrow by instead using `Additional_repositories: https://apache.r-universe.dev/` or `Additional_repositories: https://nightlies.apache.org/arrow/r`.